main.go

```diff
@@ -18,13 +18,16 @@ package main
 import (
 	"flag"
 	"fmt"
+	"io/ioutil"
 	"log"
 	"os"
+	"path/filepath"
 	"sort"
 
 	"github.com/release-engineering/backvendor/backvendor"
 )
 
+var helpFlag = flag.Bool("help", false, "print help")
```

Comment:
Seems like we can remove this and only look for `flag.ErrHelp`.

Reply:
Yes, that's what I was trying to say in the pub last night. However, it does mean that PrintDefaults then doesn't list a -help option. It could still be in the *usage: ...* string, obviously. What's your preference?